specification/metrics/metric-requirement-level.md

```diff
@@ -0,0 +1,42 @@
+# Metric Requirement Levels
+
+This document is the same as [the specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/metric-requirement-level.md).
```

Comment:
I think it would be better if this document was just part of the spec (as the spec will specify this rather than the semantic conventions for themselves) and we link to it where needed.
What would be the benefit of duplicating it here?